packages/fast-components-react-base/src/toolbar/README.md

```diff
@@ -0,0 +1,24 @@
+## Toolbar
+A generic toolbar component implementing the [toolbar](https://www.w3.org/TR/wai-aria-1.1/#toolbar) role and the keyboard navigation aligning to that role. The *toolbar* will accept any child elements but will only assign focus to children with roles specified in the component's "focusableRoles" prop.
+
+### Usage
+To accurately manage document focus, the *toolbar* assumes that all only children with the appropriate roles should be focusable. It also expects that these elements are passed as *direct* children. Wrapping all menu items inside of a `React.Fragment` (or some other single element) will throw off the internal focus tracking of the *toolbar*. Additionally, the *toolbar* expects that all menu item children implement the `tabIndex` property and apply that property to the underlying DOM.
+
+The default list of valid roles for toolbar items is as follows:
+    "button",
+    "checkbox",
+    "link",
+    "menuitem",
+    "menuitemradio",
+    "menuitemcheckbox",
+    "progressbar",
+    "radio",
+    "searchbox",
+    "slider",
+    "spinbutton",
+    "switch",
+    "textbox",
```

Comment:
Can you link to where this is coming from? I can't seem to find the reference here: https://w3c.github.io/aria-practices/#toolbar

Reply:
It's a first crack at a default list of widget roles that would make sense, not based on any toolbar spec but derived by looking at widget roles - https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques

Reply:
From the WAI Aria spec lined above:
> Authors MAY manage focus of descendants for all instances of this role

We also need a way of opting out of focus trapping. The key word here is "may".

Reply:
the second iteration of the component takes a different approach, new pr coming soon